lib/bundle/core+track+encrypt.js

```diff
@@ -5,8 +5,13 @@ import { Encrypt } from '../encrypt';
  /**
   * @param {object} [cfg]
   */
-export default function Ravelin(cfg) {
+function Ravelin(cfg) {
+  cfg.Promise = Ravelin.Promise;
```

Comment:
Does this need to be:

```suggestion
  cfg.Promise = cfg.Promise || Ravelin.Promise;
```